packages/notifi-react-card/lib/components/subscription/EventTypeHealthCheckRow.tsx

```diff
@@ -1,5 +1,15 @@
 import { ThresholdDirection } from '@notifi-network/notifi-core';
+import {
+  EventTypeItem,
+  HealthCheckEventInputsWithCustomPercentage,
+  HealthCheckEventInputsWithIndex,
+  HealthCheckInputs,
+} from '@notifi-network/notifi-frontend-client';
 import clsx from 'clsx';
+import {
+  subscribeAlertByFrontendClient,
+  unsubscribeAlertByFrontendClient,
+} from 'notifi-react-card/lib/utils/frontendClient';
```

Comment:
similar as on the other PR, I think I prefer relative import

Reply:
Issue addressed!